src/actions/index.js

```diff
@@ -356,7 +357,8 @@ function updateOutput(
     outputProp,
     getState,
     requestUid,
-    dispatch
+    dispatch,
+    changedProps,
```

Comment:
Per our slack discussion this morning, I guess change this to `changedPropIds`?

Reply:
Yes, also filtering the inputs that are not connected.